.circleci/config.yml

```diff
@@ -61,22 +85,15 @@ commands:
       - run:
           name: 'Maybe Gracefully Halt'
           command: /tmp/restored-workspace/maybe_gracefully_halt.sh
-      - restore_cache:
-          keys:
-            - source-v1-{{ .Branch }}-{{ .Revision }}
-            - source-v1-{{ .Branch }}-
-            - source-v1-
-      - checkout
+      - checkout_repo
       - run:
           name: 'Configure Development Environment'
           command: |
             ./.circleci/fetch_merge_commit.sh
             ./.circleci/check_config.sh
             ./.circleci/restore_build_output.sh
             cat ./build-system/test-configs/hosts | sudo tee -a /etc/hosts
-      # TODO(danielrozenberg): consolidate the extra steps.
```

Comment:
Now that we have a reusable command called `setup_node_environment` that installs node LTS and then installs npm packages using CircleCI's official orbs, I decided to remove this TODO, and instead use the ⭐ emoji to make the main step easier to spot. WDYT?

Reply:
👍